robot/robot.go

```diff
@@ -95,8 +96,20 @@ type LocalRobot interface {
 	// StartWeb starts the web server, will return an error if server is already up.
 	StartWeb(ctx context.Context, o weboptions.Options) error
 
+	// StartLite starts the local grpc lite server.
+	StartLite(ctx context.Context) error
```

Comment:
I can (anad have) removed StartModule() from Robot, but ModuleAddress() is needed, otherwise webService() needs to become WebService() to expose ModuleAddress in turn. (It's consumed in the module manager.)